### PR TITLE
Fix labels from Terragrunt on contributing

### PR DESCRIPTION
**Pull request by marinalimeira**
+ Copy TEMPLATE from Terragrunt's repo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ We also recommend updating the automated tests *before* updating any code (see [
...
 ensures the tests stay up to date and verify all the functionality in this Module, including whatever new 
-functionality you're adding in your contribution. Check out [Developing Terragrunt](README.md#developing-terragrunt) 
+functionality you're adding in your contribution. Check out [Developing cloud-nuke](README.md#developing-cloud-nuke) 
 for instructions on running the automated tests. 
 
 ## Update the code
 
 At this point, make your code changes and use your new test case to verify that everything is working. Check out 
-[Developing Terragrunt](README.md#developing-terragrunt) for instructions on how to build and run Terragrunt locally.
+[Developing cloud-nuke](README.md#developing-cloud-nuke) for instructions on how to build and run Terragrunt locally.
```
Comment (autero1) [inline CONTRIBUTING.md]:
```suggestion
[Developing cloud-nuke](README.md#developing-cloud-nuke) for instructions on how to build and run `cloud-nuke` locally.
```

File: _docs/rfc/TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# RFC Template
+
+This is a template you can use for proposing new major features to Terragrunt. When creating a new RFC, copy this
```
Comment (autero1) [inline _docs/rfc/TEMPLATE.md]:
```suggestion
This is a template you can use for proposing new major features to `cloud-nuke`. When creating a new RFC, copy this
```

File: _docs/rfc/TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+template and fill in each respective section.
+
+**STATUS**: In proposal _(This should be updated when you open a PR for the implementation)_
+
+
+## Background
+
+This section should describe why you need this feature in Terragrunt. This should include a description of:
```
Comment (autero1) [inline _docs/rfc/TEMPLATE.md]:
```suggestion
This section should describe why you need this feature in `cloud-nuke`. This should include a description of:
```

File: _docs/rfc/TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+- The problem you are trying to solve.
+- The reason why Terraform can't solve this problem, or data points that suggest there is a long enough time horizon for
```
Comment (autero1) [inline _docs/rfc/TEMPLATE.md]: Maybe remove this bullet point?

File: _docs/rfc/TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+solutions that appear to be valid, so it helps to be explicit about why certain solutions were not chosen.
+
+
+## References
+
+This section should include any links that are helpful for background reading such as:
+
+- Relevant issues
+- Links to PRs: at a minimum, the initial PR for the RFC, and the implementation PR.
+- Links to Terragrunt releases, if the proposed solution has been implemented.
```
Comment (autero1) [inline _docs/rfc/TEMPLATE.md]:
```suggestion
- Links to `cloud-nuke` releases, if the proposed solution has been implemented.
```

File: _docs/rfc/TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+# RFC Template
```
Comment (autero1) [inline _docs/rfc/TEMPLATE.md]: Is there a link leading to the RFC template? If not, consider adding one in `CONTRIBUTING.md`

Comment (marinalimeira) [reply _docs/rfc/TEMPLATE.md]: Yes! It's on line 28.

**Comment by brikis98** [comment]
Just realized we never merged this! Merging now.

**Pull request merged**